Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuController.cs

```diff
@@ -0,0 +1,513 @@
+// Panel Menu Controller
```

Comment:
This will need a Prefabs header then, plus all of the public variables (that show up in the inspector) have a tooltip.

Best to look at this file for an example: https://github.com/thestonefox/SteamVR_Unity_Toolkit/blob/master/Assets/VRTK/Scripts/Controls/2D/RadialMenu/RadialMenu.cs